api/tests/unit/app_analytics/test_analytics_db_service.py

```diff
@@ -385,32 +385,58 @@ def test_get_feature_evaluation_data_calls_get_feature_evaluation_data_from_loca
     )
 
 
-@pytest.mark.parametrize(
-    "period",
-    [
-        CURRENT_BILLING_PERIOD,
-        PREVIOUS_BILLING_PERIOD,
-    ],
-)
-def test_get_usage_data_returns_empty_list_when_unset_subscription_information_cache(
-    period: str,
+@pytest.mark.freeze_time("2023-01-19T09:09:47.325132+00:00")
+def test_get_usage_data_returns_empty_list_when_unset_subscription_information_cache_for_previous_billing_period(
```

Comment:
I think the name of this test (and the following one) needs updating to reflect the new behaviour?

Reply:
Nice catch. Fixed.